main.css

```diff
@@ -0,0 +1,73 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto:400,900');
+@import url('https://fonts.googleapis.com/css?family=Raleway:400,600');
+
+html {
+  font-size: 16px;
+}
+body {
+  font-size: 1rem;
+  font-family: 'Roboto', sans-serif;
+}
+
+
+
+/* Programme Builder CELLS */
+
+.prog-cell {
+  border: 1px solid #979797;
+}
+  .prog-cell__action{
+    font-family: Roboto;
+    font-size: 0.6875rem; /* 11px */
+    float:right;
+    background-color: #DBDBDB;
+    box-shadow: inset 0 -1px 2px 0 rgba(0,0,0,0.5),inset 0 1px 2px 0 rgba(255,255,255,0.5);
+    border-radius: 3px;
+    border: 1px solid #808080;
+
+  }
+  .prog-cell__header { /* anchor tag */
+    display: block;
+    background-color: rgb(0, 48, 102);
+    padding: 0.25rem; /* 4px */
+    margin-bottom: 0.25rem; /* 4px */
+  }
+  .prog-cell__name {
+    font-size: 0.8125rem; /* 13px */
+    color: #ffffff;
+    text-transform: capitalize;
+    text-decoration: none;
+  }
+  .prog-cell__body {
+    padding: 0.25rem;
+  }
+  .prog-cell__data {
+    font-size: 0.6875rem; /* 11px */
+    font-family: 'Raleway', sans-serif;
+    display: block;
+    margin-bottom: 0.25rem; /* 4px */
+  }
+  .prog-cell__location {
+    text-transform: capitalize;
+  }
+  .prog-cell__chair {
+    color: #808080;
+    font-style: italic;
+    font-weight: 600;
+  }
+  .prog-cell__track {
+    font-size: 0.625rem;
+    background-color: #444444;
+    font-weight: 900;
+    color: #ffffff;
+    padding: 0.125rem;
+    float:right;
```

Comment:
Having reviewed the behaviour and layout I think we should just treat this like any other piece of data on the cells.
One exception though, to make sure the background only wraps the words of the element this should be 
`display: inline-block;`